src/includes/getting-started-install/android.okhttp.mdx

```diff
@@ -0,0 +1,12 @@
+<Note>
+
+The OkHttp integration is supported in Sentry's Android SDK version `4.4.0` and above.
+
+</Note>
+
+To add the OkHttp integration, initialize the [Android SDK](/platforms/android/), then add the `sentry-android-okhttp` dependency. Using Gradle:
+
+```groovy
+implementation 'io.sentry:sentry-android:{{ packages.version('sentry.java.android', '4.4.0') }}'
+implementation 'io.sentry:sentry-android-okhttp:{{ packages.version('sentry.java.android.okhttp', '4.4.0') }}'
```

Comment:
```suggestion
implementation 'io.sentry:sentry-android:4.4.0-alpha.1'
implementation 'io.sentry:sentry-android-okhttp:4.4.0-alpha.1'
```

we can't use `packages.version` because its an alpha and we don't release this to release registry